movietheatre_solution.py

```diff
@@ -38,3 +38,4 @@ def movie_cost(purchase_entry):
 
 movie_cost(purchase_entry)
 
+#I added to the purchase entry

```

Comment:
whats the purpose for line 3 & 4?
what is line 10 helping you with?

Reply:
> 

that variable contains all you need since you arent manipulating it

Reply:
Those got in there during my thinking process, and I forgot to take them out after the solution

Reply:
Line 10 - 12, is my way of checking through the list, what am I missing out or duplicating?

Reply:
when you `enumerate()`, you have an index and each item in the list in each iteration. e.g 
```
items = [egg, bread, beans]
for count, value in enumerate(items):
    print(count,value)
```
# 0, egg
# 1, bread
#2, beans

Reply:
you have not put the question there.

Reply:
Here in the comment section or in the code?

Reply:
Also i dont think this function is doing what it’s supposed to do. If you remove any of the item on the list of purchase_enter then the answer will be wrong.